R/pkg/R/functions.R

```diff
@@ -362,8 +357,8 @@ setMethod("cov", signature(x = "characterOrColumn"),
 
 #' @rdname cov
 #'
-#' @param col1 First column to compute cov_samp.
-#' @param col2 Second column to compute cov_samp.
+#' @param col1 the first Column object.
+#' @param col2 the second Column object.
```

Comment:
I'd say that applies to a couple of other cases in WindowsSpec.R or column.R too, but I'm ok either way.